pkgs/development/mobile/androidenv/cmake.nix

```diff
@@ -2,8 +2,8 @@
 
 deployAndroidPackage {
   inherit package os;
-  buildInputs = [ autoPatchelfHook ]
-    ++ lib.optional (os == "linux") [ pkgs.stdenv.glibc pkgs.stdenv.cc.cc ];
+  nativeBuildInputs = [ autoPatchelfHook ]
+    ++ buildInputs = lib.optional (os == "linux") [ pkgs.stdenv.glibc pkgs.stdenv.cc.cc pkgs.ncurses5 ];
```

Comment:
```suggestion
  nativeBuildInputs = [ autoPatchelfHook ];
  buildInputs = lib.optional (os == "linux") [ pkgs.stdenv.glibc pkgs.stdenv.cc.cc pkgs.ncurses5 ];
```

Reply:
To what extend have you tested the changes?

Reply:
FWIW this change is based on [my suggested patch here](https://github.com/NixOS/nixpkgs/pull/89775#issuecomment-726798100) which worked fine as far as I could tell. Just seems like the patch got mangled on its way into the PR somehow.

Reply:
Ok. Maybe some bug introduced with a merge.

Reply:
Looks like that. Don't know how that happened.